spec/services/contents/analyzer_spec.rb

```diff
@@ -3,8 +3,9 @@
 require 'rails_helper'
 
 RSpec.describe Contents::Analyzer do
-  let(:content) { create(:content) }
+  include WorkMappingFixtures
```

Comment:
Why?

Reply:
Good catch. This itself is dead code because I wound up removing the test coverage I added for `Contents::Analyzer` which was flappy. Will remove this.